vendor/vendor-android/ddmlib/src/main/kotlin/com/malinskiy/marathon/android/ddmlib/DdmlibAndroidDevice.kt

```diff
@@ -301,11 +313,18 @@ class DdmlibAndroidDevice(
             prepareRecorderListener(feature, fileManager, devicePoolId, attachmentProviders)
         } ?: NoOpTestRunListener()
 
+        val vrtListener = if (configuration.isPullScreenshotEnabled()) {
```

Comment:
rename `vrtListener` to `pullScreenshotsListener`?

Reply:
Yes, also renamed package